agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4741,6 +4744,42 @@ bool slave_thread::handle_ap_capability_query(Socket *sd, ieee1905_1::CmduMessag
     return true;
 }
 
+bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received CLIENT_STEERING_REQUEST_MESSAGE , mid=" << std::hex << int(mid);
+    //TODO: validate the BTM request before sending the ack.
```

Comment:
I think the minimum validation we should to is to check the BSSID. Indeed, both slaves are handling this request, so both of them will send an Ack and a BTM Report. This is clearly wrong.

However, it doesn't matter for certification, and this will anyway be rewritten extensively when we complete the agent implementation, so (nitpick).